_config.yml

```diff
@@ -17,8 +17,13 @@ bf:
  version: 6.1.0
 
 omero:
- version: 5.4.10
- build: b105
+ version: 5.5.0-rc2
```

Comment:
rc3 was merged this morning, right? Does this need bumping?

Reply:
I will do a final bump after 5.50. This was mainly to test the structure